doc/architecture/decisions/0003-image-synching.md

```diff
@@ -0,0 +1,47 @@
+# 3. Image synching
+
+Date: 2024-07-12
+
+## Status
+
+Accepted
+
+## Context
+
+EGI provides an image repository (AppDB) for users to share VM images and VOs to
+add them to a list of images to be available at the sites supporting the VO.
+This has been managed with site-level although the uploading of images is for
+most sites a user-level operation that does not need any special privileges.
```

Comment:
This last sentence is not clear to me.
* `This has been managed with site-level although` seems incomplete
* Uploading of images does require special privileges at the cloud site, right?

Reply:
Updated the text, broken sentence before.
Uploading in default OpenStack does not require privileges, but some EGI sites restrict it. If sites start seeing this as valuable and do not want to widely open the upload of images, we can always create a role for this and ask for mapping it properly 😬 
Please check that it now makes sense.